src/main/java/train/common/core/handlers/WorldEvents.java

```diff
@@ -129,13 +129,13 @@ public void entityJoinWorldEvent(EntityJoinWorldEvent event) {
 				if (event.entity.getUniqueID().equals(UUID.fromString(entry[0]))) {
 					if (entry[1].equals("0")) {
 						throw new ReportedException(CrashReport.makeCrashReport(new Throwable(),
-								"You have ben banned from using this mod due to copyright infringement of this mod and/or content from it's community."));
+								"You have been banned from using this mod due to copyright infringement of this mod and/or content from its community."));
 					} else if (entry[1].equals("2")) {
 						throw new ReportedException(CrashReport.makeCrashReport(new Throwable(),
 								"This is childish. I'm out."));	
 					} else {//1
 						throw new ReportedException(CrashReport.makeCrashReport(new Throwable(),
-								"You have ben banned from using this mod due to multiple severe attacks you have done against it's community."));
+								"You have been banned from using this mod due to multiple severe attacks you have done against its community."));
```

Comment:
I feel this would read better as:
```suggestion
								"You have been banned from using this mod due to your multiple severe attacks against its community."));
```